cdqa/pipeline/cdqa_sklearn.py

```diff
@@ -177,6 +177,55 @@ def predict(self, X=None, return_logit=False):
             raise TypeError("The input is not a string or a list. \
                             Please provide a string or a list of strings as input")
 
+    def n_predictions(self, X=None, return_logit=False):
```

Comment:
Instead of creating another method for this, I think it would be best to do it as an option inside the `predict` method. We would like to be as most compliant with sklearn as we can, so creating another method for that would break this compliance. Moreover, there will be a lot of repeated code.

My advice is: add an argument to the `predict` method (maybe call it`n_predictions`) with default value `None`. It should be an integer >= 1 or `None`. If `None` we do the standard prediction, if its an integer we perform the "`n_predictions`" method.

We should document it as well in the method docstring, so that users can understand its behavior.

Reply:
On it!

Reply:
Hey @andrelmfarias,
Thanks for the review.
I have incorporated your changes and wrote a test for it.
Please let me know of further improvements needed.